apps/www/pages/blog/[slug].tsx

```diff
@@ -29,16 +72,21 @@ export async function getStaticPaths() {
   }
 }
 
-export async function getStaticProps({ params }: any) {
-  const filePath = `${params.slug}`
+export const getStaticProps: GetStaticProps<BlogPostPageProps, Params> = async ({ params }) => {
+  const filePath = `${params?.slug}`
```

Comment:
Perhaps it'd be better to `throw` or return empty if `params` was undefined vs. silently fail.

Reply:
Done, I decided to `throw`